### ISSUE TITLE
Consider Removing Vertical Alignment for sub and sup

### ISSUE THREAD

**Issue by ars265** [dhg/Skeleton#102]
Do to the base.css adding the vertical-align: baseline; for both sub and sup elements, their default behavior is no longer. I suggest either normalizing these or removing the vertical alignment behavior.

**Comment by jamiebuilds** [comment]
From [`normalize.css`](https://github.com/necolas/normalize.css/blob/master/normalize.css):

```
sub,
sup {
    font-size: 75%;
    line-height: 0;
    position: relative;
    vertical-align: baseline;
}

sup {
    top: -0.5em;
}

sub {
    bottom: -0.25em;
}​
```

https://github.com/necolas/normalize.css/blob/master/normalize.css#L183

**Comment by dhg** [comment]
Resolved with the release of V2.0 and the inclusion of normalize.css :) As always though, lemme know if I missed anything in closing this.
